### PR TITLE
VIH-7857  - non unique page title

### PR DESCRIPTION
**Pull request by abbasiTahir**
https://tools.hmcts.net/jira/browse/VIH-7857

added unique title age

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts [hunk 1]
```diff
@@ -497,6 +502,27 @@ export abstract class WaitingRoomBaseDirective {
...
+            if (room.includes('JudgeConsultationRoom')) {
+                title = 'Video Hearings - JOH Consultation Room';
+            } else if (room.includes('JudgeJOHConsultationRoom')) {
+                title = 'Video Hearings - JOH Consultation Room';
+            } else if (room.includes('ConsultationRoom')) {
+                title = 'Video Hearings - Private Consultation Room';
+            } else if (room.includes('HearingRoom')) {
+                title = 'Video Hearings - Hearing Room';
+            } else {
+                title = 'Video Hearings - Waiting room';
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:519]: could we set this as the default value to reduce the number of if else conditions

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts [hunk 2]
```diff
@@ -497,6 +502,27 @@ export abstract class WaitingRoomBaseDirective {
         );
     }
 
+    private setTitle(roomTransfer: RoomTransfer): void {
+        const room: string = roomTransfer.to_room;
+        const transferId = roomTransfer.participant_id;
+        if (this.participant.id === transferId) {
+            let title;
+            if (room.includes('JudgeConsultationRoom')) {
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:510]: combine this with JudgeJOHConsultationRoom as they are setting the same title

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:519]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts (hunk 2 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:510]: done

Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts [hunk 3]
```diff
@@ -497,6 +502,27 @@ export abstract class WaitingRoomBaseDirective {
         );
     }
 
+    private setTitle(roomTransfer: RoomTransfer): void {
+        const room: string = roomTransfer.to_room;
+        const transferId = roomTransfer.participant_id;
+        if (this.participant.id === transferId) {
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts]: test added

**Review by will-craig — APPROVED** [review]

**Review by omio-ma — APPROVED** [review]

**Pull request merged**